base/Dockerfile

```diff
@@ -40,4 +40,9 @@ RUN chmod 755 /usr/local/bin/go-wrapper
 RUN ln -s /go/src/app /app
 WORKDIR /go/src/app
 
+ENV CLOUD_DEBUG_ENABLE "true"
```

Comment:
Why does this env variable need to be set? Is there an option to turn it off? Why?